pkg/controller/jobframework/base_webhook_test.go

```diff
@@ -246,7 +246,9 @@ func TestBaseWebhookDefault(t *testing.T) {
 	for name, tc := range testcases {
 		t.Run(name, func(t *testing.T) {
 			ctx, _ := utiltesting.ContextWithLog(t)
-			features.SetFeatureGateDuringTest(t, features.LocalQueueDefaulting, tc.localQueueDefaulting)
+			if !tc.localQueueDefaulting {
```

Comment:
I would prefer to revert all these changes and keep the old code. 

ie

```suggestion
			features.SetFeatureGateDuringTest(t, features.LocalQueueDefaulting, tc.localQueueDefaulting)
```

Apply that to all the tests below also.